src/transformers/utils/import_utils.py

```diff
@@ -44,26 +62,83 @@
 
 FORCE_TF_AVAILABLE = os.environ.get("FORCE_TF_AVAILABLE", "AUTO").upper()
 
+# This is the version of torch required to run torch.fx features and torch.onnx with dictionary inputs.
+TORCH_FX_REQUIRED_VERSION = version.parse("1.10")
+
+
+_accelerate_available, _accelerate_version = _is_package_available("accelerate", return_version=True)
+_apex_available = _is_package_available("apex")
+_bitsandbytes_available = _is_package_available("bitsandbytes")
+_bs4_available = _is_package_available("bs4")
+_coloredlogs_available = _is_package_available("coloredlogs")
+_datasets_available = _is_package_available("datasets")
+_decord_available = importlib.util.find_spec("decord") is not None
+_detectron2_available = _is_package_available("detectron2")
+_diffusers_available = _is_package_available("diffusers")
```

Comment:
We don't have this check currently on main.

Reply:
they have been introduced with #23214. should we remove them?
btw. I haven't checked that all "is_available" are used. just refactored assuming they are

Reply:
The private variables should have been cleaned up, this is a mistake indeed.